onnx2torch/node_converters/resize.py

```diff
@@ -58,7 +58,7 @@ def forward(
             sizes: Optional[torch.Tensor] = None,
     ) -> torch.Tensor:
         torch_mode = _onnx_mode_to_torch_mode(self.onnx_mode, input_tensor.dim() - 2)
-        if roi is not None and roi.nelement() != 0:
+        if roi is not None and roi.nelement() != 0 and roi.nelement() != (roi==1).sum():
```

Comment:
Because if we use paddle upsample op and export onnx. we will get a Resize op with roi attr, it is a tensor which is filled with 1. So maybe ones tensor roi mean nothing to do with resize

Reply:
According [specification](https://github.com/onnx/onnx/blob/main/docs/Operators.md#Resize) roi input can be ignored in all `coordinate_transformation_mode` except `"tf_crop_and_resize"`.  Therefore we should check roi input only for `"tf_crop_and_resize"` mode. You can add `check_roi` flag to `__init__` and fix condition:
```python
if self._check_roi and roi is not None and roi.nelement() != 0:
    ...
```

Reply:
Got it I will fix